source/common/upstream/eds.h

```diff
@@ -41,6 +41,7 @@ class EdsClusterImpl : public BaseDynamicClusterImpl,
   std::unique_ptr<Config::Subscription<envoy::api::v2::ClusterLoadAssignment>> subscription_;
   const LocalInfo::LocalInfo& local_info_;
   const std::string cluster_name_;
+  const std::string local_cluster_name_;
```

Comment:
nit: Would it be better to store a const reference to the ClusterManager here and have the localCluster() method be const on that? Not a big deal but thought I would mention it.